manifests/alertmanager.pp

```diff
@@ -157,10 +157,10 @@
   }
 
   $alertmanager_reload = $prometheus::init_style ? {
-    'systemd'          => "systemctl reload-or-restart ${service_name}",
-    /^(upstart|none)$/ => "service ${service_name} reload",
-    /^(sysv|sles)$/    => "/etc/init.d/${service_name} reload",
-    'launchd'          => "launchctl stop ${service_name} && launchctl start ${service_name}",
+    'systemd'              => "systemctl reload-or-restart ${service_name}",
+    /^(upstart|none)$/     => "service ${service_name} reload",
+    /^(sysv|redhat|sles)$/ => "/etc/init.d/${service_name} reload",
```

Comment:
that doesn't look correct to me, shouldn't they use systemd?

Reply:
This was part of a 'too violent rebase'.